BlackNectar/BlackNectar/StoresTableViewController.swift

```diff
@@ -380,3 +382,22 @@ fileprivate extension StoresTableViewController {
     
 }
 
+//MARK - Load User Preferences Code
+fileprivate extension StoresTableViewController {
+    
+    func loadDefaults() {
+        
+        if defaults.double(forKey: "distanceFilter") > 0 {
+            
+            showStores = defaults.bool(forKey: "isStore")
```

Comment:
There is definitely some code duplication going on here.
It seems like you may want to capture the idea of user's settings in a separate class, and have these two view controllers call that instead.